lib/htmlbars-minifier.js

```diff
@@ -1,18 +1,25 @@
-const Filter = require('broccoli-filter');
+const Filter = require('broccoli-persistent-filter');
+const md5Hex = require('broccoli-persistent-filter/lib/md5-hex');
 
-function HtmlbarsMinifier(inputTree, options) {
+function HtmlbarsMinifier(inputNode, options) {
```

Comment:
Try to migrate this to ES6 classes and add the property `annotation` to the plugin (debugging purposes). The code seems to be easier to understand using classes:

```js
class Thumbnail extends Filter {
  constructor(inputNode, options = {}) {
    options = Object.assign(options || {}, {
      persist: true,
      annotation: 'HTMLBars Minifier',
    });
    super(inputNode, options);
  }

  // TODO...
}
```

Reply:
I'm going to open a new issue with this purpose. Thanks!

Reply:
Finally I migrated to ES6 classes, and resolved the `persist=true` issue